src/main/java/com/iota/iri/BundleValidator.java

```diff
@@ -51,13 +95,15 @@
                         instanceTransactionViewModels.get(0).setValidity(tangle, -1);
```

Comment:
Is it worth adding a comment above that this is where we do the semantic checks?